.ci/prepare_env_utils.sh

```diff
@@ -67,7 +68,16 @@ function checkAllCreds() {
 	set +x
 }
 
-
-
-
-
+# JOB_TYPE=che-functional-tests|functional-tests|periodic-tests
+function archiveArtifacts() {
+  echo "Archiving artifacts from ${DATE} for ${JOB_NAME}/${BUILD_NUMBER}"
+  ls -la ./artifacts.key
+  chmod 600 ./artifacts.key
+  chown $(whoami) ./artifacts.key
+  mkdir -p ./${JOB_TYPE}/${JOB_NAME}/${BUILD_NUMBER}/surefire-reports
+  #cp -R ./logs/artifacts/surefire-reports/*.txt ./${JOB_TYPE}/${JOB_NAME}/${BUILD_NUMBER}/surefire-reports
+  cp ./logs/*.log ./${JOB_TYPE}/${JOB_NAME}/${BUILD_NUMBER}/
+  cp -R ./logs/artifacts/screenshots/ ./${JOB_TYPE}/${JOB_NAME}/${BUILD_NUMBER}/
+  #cp -R ./logs/artifacts/videos/ ./${JOB_TYPE}/${JOB_NAME}/${BUILD_NUMBER}/
```

Comment:
I don't see any advantage of this comment - as far as I understand we are not planning to add videos to tests.